app/web/src/store/func/funcs.store.ts

```diff
@@ -318,6 +322,9 @@ export const useFuncStore = () => {
               { componentId },
               { viewId },
             ]),
+            optimistic: (requestUlid) => {
+              this.pendingManagementRunRequestUlid = requestUlid;
```

Comment:
what happens if there are multiple in flight requests?

Reply:
Last one wins. If the order is bonked, where the latest execution's event arrives before the previous execution's event (because of thread scheduling or because one operator is a lot more complex than another), then the selection just won't happen. I think that's ok, since I'm not sure what the ideal solution is there: we don't want it to flap between selections if a bunch arrive at once.